index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="/styles.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+   
+    <div id= "main">
+     <section id="title">
+       <h1 class = "h1">Dr. Norman Borlaug</h1>
+       <p  class = "p">The man who saved a billion lives</p>
+     </section>
+     <section id="img-div">
+       <img id="image" src="image/tribute-page-main-image.jpeg">
```

Comment:
- [ ] Please add the `alt` text of the Tribute page main image so as to improve accessibility and to help visually impaired people to access your page. See my suggestions below.

```suggestion
       <img id="image" src="image/tribute-page-main-image.jpeg" alt="Main page image" />
```